server/core/runtime/env_step_runner_test.go

```diff
@@ -36,6 +37,13 @@ func TestEnvStepRunner_Run(t *testing.T) {
 			Value:    "test",
 			ExpValue: "test",
 		},
+		{
+			Command: "echo $COMMENT_ARGS",
+			Envs: map[string]string{
+				"COMMENT_ARGS": "\\t\\e\\s\\t,\\v\\1,\\v\\2,\\v\\3",
+			},
+			ExpValue: "\\t\\e\\s\\t,\\v\\1,\\v\\2,\\v\\3",
```

Comment:
run step uses `COMMENT_ARGS` with escaped comment args like [this](https://github.com/runatlantis/atlantis/blob/fdcb1adfb064ea71a33f880f3bc3b1ffad09563e/server/core/runtime/run_step_runner.go#L43).

But when using env step, this test case fail with following result.

```
        exp: (string) (len=23) "\\t\\e\\s\\t,\\v\\1,\\v\\2,\\v\\3"
        ******
        got: (string) (len=18) "\t\\e\\s\t,\v\\1,\v\\2,\v\\3"
```

Reply:
ah, `\t` is handled as control character...?

Reply:
This works fine.

```go
		{
			Command: "printf '%s' $COMMENT_ARGS",
			Envs: map[string]string{
				"COMMENT_ARGS": "\\t\\e\\s\\t,\\v\\1,\\v\\2,\\v\\3",
			},
			ExpValue: "\\t\\e\\s\\t,\\v\\1,\\v\\2,\\v\\3",
		},
```